components/provisioner/internal/api/resolver_integration_with_gardener_test.go

```diff
@@ -216,7 +221,7 @@ func TestProvisioning_ProvisionRuntimeWithDatabase(t *testing.T) {
 
 			testProvisionRuntime(t, ctx, resolver, fullConfig, config.runtimeID, shootInterface, secretsInterface, config.auditLogConfig)
 
-			testUpgradeGardenerShoot(t, ctx, resolver, dbsFactory, config.runtimeID, config.upgradeShootInput, shootInterface, inputConverter)
+			//testUpgradeGardenerShoot(t, ctx, resolver, dbsFactory, config.runtimeID, config.upgradeShootInput, shootInterface, inputConverter)
```

Comment:
Why did  commented out this test? If this is intentional the function `testUpgradeGardenerShoot` should also be removed.

Reply:
Code restored